SONiC_workgroup.md

```diff
@@ -0,0 +1,28 @@
+# SONiC Workgroups                    
+
+| Workgroups | Mailing List | Status | Workgroup Charter | Workgroup Meeting |
+| -----------| ------------ | ------ | ----------------- | ----------------- |
+| [Testing Workgroup](https://groups.google.com/forum/#!forum/sonic-test-workgroup)  | sonic-test-workgroup@gmail.com   | Active  | 1.  Testbed improvement. <br> 2.  Pytest framework enhancement <br> 3.  Pytest conversion from ansible test. <br>  4.  Specific/new tests <br> 5.  Visualized DUT with data plane capabilities <br> 6.  One(AKA, Crystalnet) integration (open source) <br> For detailed SONiC test-workgroup charter click [here](https://groups.google.com/forum/#!topic/sonic-test-workgroup/OD_UhjqlZ9Y). |  |
+| [Management Workgroup](https://groups.google.com/forum/#!forum/sonic-mgmt-workgroup)  | sonic-mgmt-workgroup@googlegroups.com   | Active  | 1.  Simplify and Standardize SONiC management <br> 2.  Make SONiC management DevOps friendly <br> 3.  Industry standard CLI for SONiC config validations and error feedback  |     |
```

Comment:
The above workgroup is no longer active. The current active group is called UMF Workgroup, mailing alias umf-working-group@lists.sonicfoundation.dev

Reply:
@adyeung Are you referring to line no 6 - Management workgroup ?

Reply:
Correct, the comment is referring to management workgroup

Reply:
@adyeung Have updated the management group as inactive and added the UMF working group. Also updated  the table for alphabetical order.